nixos/modules/services/x11/display-managers/lightdm.nix

```diff
@@ -72,17 +40,17 @@ let
       ${cfg.extraSeatDefaults}
     '';
 
-  gtkGreeterConf = writeText "lightdm-gtk-greeter.conf"
-    ''
-    [greeter]
-    theme-name = Adwaita
-    icon-theme-name = Adwaita
-    background = ${cfg.background}
-    '';
-
 in
 {
+  # Note: the order in which lightdm greeter modules are imported
+  # here determines the default: later modules (if enable) are
+  # preferred.
```

Comment:
Ah right, I'm not entirely sure if it's going to override without specifying the `type` property on the `lightdm.greeter` option. It is the type of the option that specifies the behavior of combining multiple values. Check out `lib/types.nix` for a list of all possible types.

I don't think it's important that this multiple-greeter stuff necessarily works right now though, because we only have Gtk anyway.